cdap-app-fabric/src/main/java/co/cask/cdap/config/DefaultConfigStore.java

```diff
@@ -44,8 +31,6 @@
  * "
  */
 public class DefaultConfigStore implements ConfigStore {
```

Comment:
I think we can remove the ConfigStore interface and just have this class named ConfigStore. The interface only have 1 implementation and it adds some complexity in guice injection.

Reply:
I'd rather keep this as is - this seems like a common pattern throughout the CDAP codebase. If this is something we wish to change we should file a JIRA.

Reply:
yea some of them are useful when we need to have different implementations in some other environment or unit test, I think you can file a JIRA to remove the interface. I think in this pr you can just rename `DefaultConfigTable` to `ConfigTable` since it is a newly added class

Reply:
renamed